docs/API/REST-V2-Usage.md

```diff
@@ -1213,10 +1213,9 @@ start as query parameters to support legacy behavior).
 |------------------|--------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | skip             | int    | Optional. Number of versions to skip.                                                                                                                                                                                                                                                                        |
 | limit            | int    | Optional. The number of versions to be returned per page of pagination. Defaults to 20.                                                                                                                                                                                                                      |
-| revision_start   | int    | Optional. The version order number to start at, for pagination.                                                                                                                                                                                                                                              |
-| revision_end     | int    | Optional. The version order number to end at, for pagination.                                                                                                                                                                                                                                                |
-| start_time_str   | string | Optional. Timestamp to start looking for applicable versions.                                                                                                                                                                                                                                                |
-| end_time_str     | string | Optional. Timestamp to stop looking for applicable versions.                                                                                                                                                                                                                                                 |
+| start   | int    | start is deprecated. It was previously used as the version order number to start at for pagination. Use revision_start instead but keep in mind that it uses lte and not lt.                                                                                                                                                                                                                                                 |
```

Comment:
We can't mark this as deprecated if it'll be used for pagination and the others cannot, since we aren't suggesting that users stop using pagination for this route. That being said, do we need to give revision start? The user is just asking for revision_end right?

Reply:
I switched it to use revision_start for pagination and not allow combining revision_start or revision_end with start. I
could just add revision_end to avoid over engineering.

Reply:
(I realize I forgot to update the documentation)